build/publisher.go

```diff
@@ -182,6 +249,32 @@ func main() {
 			tls.TLS_ECDHE_RSA_WITH_CHACHA20_POLY1305,
 		},
 	}
+
+	if tlsBool {
+		httpServerCertStore, err := newCertStore(serverCert, serverKey)
+		if err != nil {
+			zlog.Fatal().Err(err).Msg("Unable to load x509 HTTP server cert from disk")
+			os.Exit(1)
```

Comment:
Dont think we need os.Exit(1) here as zlog.Fatal() should stop the program via os.Exit(1) for us.

Reply:
Fixed, thanks.